### PR TITLE
Added scripts to install identity and auth standalone, with a script …

### PR DESCRIPTION
**Pull request by michaelvidal24**
…to register the patient safety application.

### CODE REVIEW

File: safety/Install-Fabric.ps1 [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        $identityDb = $userEnteredIdentityDb
+    }
+
+    $authorizationDb = "Authorization"
+    $userEnteredAuthorizationDb = Read-Host  "Enter the Authorization db name or accept the default [$authorizationDb]"
+    Write-Host ""
+    if(![string]::IsNullOrEmpty($userEnteredAuthorizationDb)){   
+        $authorizationDb = $userEnteredAuthorizationDb
+    }
+
```
Comment (bsmithvt) [inline safety/Install-Fabric.ps1:29]: `datbase` typo

File: safety/Install-Fabric.ps1 [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    Write-Host "Deploying Fabric.Identity datbase..."
+    .$dacPacPath /a:Publish /Profile:"Fabric.Identity.SqlServerIncludingTables.publish.xml" /SourceFile:"Fabric.Identity.SqlServer.dacpac" /TargetServerName:$databaseServer /TargetDatabaseName:$identityDb
+    Write-Host ""
+
```
Comment (bsmithvt) [inline safety/Install-Fabric.ps1]: `datbase` typo

File: safety/Register-Patient-Safety.ps1 [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+	throw $_.Exception
+}
+
+
+$userEnteredFabricInstallerSecret = Read-Host  "Enter the Fabric Installer Secret or hit enter to accept the default [$fabricInstallerSecret]"
+Write-Host ""
+if(![string]::IsNullOrEmpty($userEnteredFabricInstallerSecret)){   
+     $fabricInstallerSecret = $userEnteredFabricInstallerSecret
+}
+
```
Comment (bsmithvt) [inline safety/Register-Patient-Safety.ps1:163]: Do we want to throw an error if `$fabricInstallerSecret, $identityServerUrl, or $authorizationSerivceURL` is empty after the user input is read?

File: safety/Register-Patient-Safety.ps1 [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+}
+Import-Module -Name .\Fabric-Install-Utilities.psm1 -Force
+
+if(!(Test-IsRunAsAdministrator))
+{
+    Write-Error "You must run this script as an administrator. Halting configuration." -ErrorAction Stop
+}
+
+$installSettings = Get-InstallationSettings "registration"
+$fabricInstallerSecret = $installSettings.fabricInstallerSecret
```
Comment (bsmithvt) [inline safety/Register-Patient-Safety.ps1:120]: Typo `authorizationSerivceURL` (on the left and right side of the assignment)

File: safety/Register-Patient-Safety.ps1 [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+$authorizationSerivceURL =  $installSettings.authorizationSerivce
```
Comment (bsmithvt) [inline safety/Register-Patient-Safety.ps1]: Do we want to call this `identityServiceUrl` to be consistent w/ the auth variable name?

**Review by bsmithvt — APPROVED** [review]

File: safety/Register-Patient-Safety.ps1 (hunk 5 above)
Comment (michaelvidal24) [reply safety/Register-Patient-Safety.ps1]: Fixing this.

File: safety/Register-Patient-Safety.ps1 (hunk 4 above)
Comment (michaelvidal24) [reply safety/Register-Patient-Safety.ps1:120]: Fixing this.

File: safety/Install-Fabric.ps1 (hunk 2 above)
Comment (michaelvidal24) [reply safety/Install-Fabric.ps1]: Fixing this.

File: safety/Install-Fabric.ps1 (hunk 1 above)
Comment (michaelvidal24) [reply safety/Install-Fabric.ps1:29]: Fixing this.

File: safety/Register-Patient-Safety.ps1 (hunk 3 above)
Comment (michaelvidal24) [reply safety/Register-Patient-Safety.ps1:163]: Good call, I'll add in that check.

**Pull request merged**